src/Integration.Vsix.UnitTests/InfoBar/InfoBarManagerTests.cs

```diff
@@ -60,13 +60,27 @@ public void InfoBarManager_AttachInfoBar_ArgChecks()
             var testSubject = new InfoBarManager(this.serviceProvider);
 
             // Simple checks
-            Exceptions.Expect<ArgumentNullException>(() => testSubject.AttachInfoBar(Guid.Empty, null, "button text", KnownMonikers.EventError));
-            Exceptions.Expect<ArgumentNullException>(() => testSubject.AttachInfoBar(Guid.Empty, "", "button text", KnownMonikers.EventError));
-            Exceptions.Expect<ArgumentNullException>(() => testSubject.AttachInfoBar(Guid.Empty, "message", null, KnownMonikers.EventError));
-            Exceptions.Expect<ArgumentNullException>(() => testSubject.AttachInfoBar(Guid.Empty, "message", " ", KnownMonikers.EventError));
+            Exceptions.Expect<ArgumentNullException>(() => testSubject.AttachInfoBar(Guid.Empty, null, KnownMonikers.EventError));
```

Comment:
The interface had two methods called _AttachInfoBar_, one of which required button text, the other of which doesn't.
Only one of the methods was tested. I've renamed one of the methods and added tests for both.

Most of the other changes in the this commit are due to the renaming.